lib/gc.cpp

```diff
@@ -34,6 +34,7 @@ limitations under the License.
 #include "log.h"
 #include "n4.h"
 
+// One can disable the GC, e.g., to run under Valgrind, by editing config.h
```

Comment:
```suggestion
// One can disable the GC, e.g., to run under Valgrind, by editing config.h or toggling
// -DENABLE_GC=OFF in CMake.
```

Reply:
Was about to suggest this change I have in another PR, but looks like it is already merged.

Reply:
I will add your suggestion, but if you prefer to just move the comment in your PR, let me know and I will close this one.

Reply:
No, it will take a while until that PR is merged.